code/modules/power/tesla/energy_ball.dm

```diff
@@ -95,6 +95,12 @@
 	else if(orbiting_balls.len)
 		dissipate() //sing code has a much better system.
 
+	else if(energy > 9000)
```

Comment:
Technically, this should be above `if(energy >= energy_to_raise)` or this will never run.